backend/upload.js

```diff
@@ -4,46 +4,75 @@ import uuidv1 from 'uuid/v1'
 import nconf from './config'
```

Comment:
I think this file should be moved to `backend/routes/upload.js`, which allows us to use the Router module in combination with a small edit of `backend/routes/index.js`. As a bonus, we can leave `backend/server.js` fairly pristine.

Router is pretty clean to use:

```javascript
const router = Router()

router.route('/path/to/thing')
  .get(
    middlewareFunc, // probably your uploadAuth function goes here
    (req, res, next) => imageUpload( /* ... */ )
  )

export default router
```